src/components/Common/SearchByMultipleFields.tsx

```
@@ -84,6 +84,9 @@ const SearchByMultipleFields: React.FC<SearchByMultipleFieldsProps> = ({
         .some((element) => clearSearch.params?.includes(element));
       clearinput ? setSearchValue("") : null;
       inputRef.current?.focus();
+      if (clearSearch?.params && clearSearch?.params.includes("search")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)